db/migrate/20180607134817_add_network_router_id_to_floating_ip.rb

```diff
@@ -0,0 +1,5 @@
+class AddNetworkRouterIdToFloatingIp < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :floating_ips, :network_router, :type => :bigint, index: true, foreign_key: true
```

Comment:
We don't use any foreign key constraints so can you remove that?  Otherwise looks good to me.